eo-runtime/src/main/java/org/eolang/Dataized.java

```diff
@@ -55,6 +55,12 @@ public final class Dataized {
      */
     private static final ThreadLocal<Integer> LEVEL = ThreadLocal.withInitial(() -> 0);
 
+    /**
+     * Max dataization level.
+     */
+    private static final ThreadLocal<Integer> MAX_LEVEL = ThreadLocal.withInitial(() ->
+            Integer.getInteger("max.dataization.log", 3));
```

Comment:
What's the point of this variable being `ThreadLocal` if it can be set only at startup time?

Reply:
@andreoss I decided that there might be a desire to have different log levels in different threads.